CMakeLists.txt

```diff
@@ -85,15 +85,6 @@ if (QUERY_MODE STREQUAL "DEVICE" AND USE_RMM)
     set_target_properties(rmm PROPERTIES IMPORTED_LOCATION ${CMAKE_BINARY_DIR}/lib/librmm.so)
     add_dependencies(rmm project_rmm)
 endif ()
-######################################################
-# CUDF
```

Comment:
is this intentional or you forgot to rebase?

Reply:
Yep, this is weird, I did rebase. Let me revert it back